development/github-desktop/github-desktop.SlackBuild

```diff
@@ -27,7 +27,7 @@ cd $(dirname $0) ; CWD=$(pwd)
 PRGNAM=github-desktop
 SRCNAM=GitHubDesktop
 VERSION=${VERSION:-3.3.6}
-BUILD=${BUILD:-2}
+BUILD=${BUILD:-3}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
 
```

Comment:
`ar x $CWD/${SRCNAM}-linux-$DEBARCH-${VERSION}-linux2.deb data.tar.xz`
further needs to change to 
`ar x $CWD/${SRCNAM}-linux-$DEBARCH-${VERSION}-linux3.deb data.tar.xz`?

Reply:
Gah, yes and I should have updated  the README as well. I'll push the fix in a moment.

Reply:
pushed fix!